releases/release-1.19/release-notes-draft.md

```diff
@@ -126,7 +126,8 @@ The new storage capacity tracking alpha feature is known to be affected by a lim
 
 ### (No, really, you MUST read this before you upgrade)
 
-- ACTION REQUIRED : Switch core master base images (kube-controller-manager) from debian to distroless. If you need Flex Volumes support using scripts, please build your own image with required packages (like bash) ([#91329](https://github.com/kubernetes/kubernetes/pull/91329), [@dims](https://github.com/dims)) [SIG Cloud Provider, Release, Storage and Testing]
+- ACTION REQUIRED: Switch core master base images (kube-controller-manager) from debian to distroless. If you need Flex Volumes support using scripts, please build your own image with required packages (like bash) ([#91329](https://github.com/kubernetes/kubernetes/pull/91329), [@dims](https://github.com/dims)) [SIG Cloud Provider, Release, Storage and Testing]
```

Comment:
Can you please open an PR to k/k to reflect the changes in the release notes there as well? Otherwise they will not sync up.